static/src/jspm-config.js

```diff
@@ -0,0 +1,135 @@
+System.config({
+  "baseURL": "/assets",
+  "transpiler": "babel",
+  "paths": {
+    "*": "javascripts/projects/*.js",
+    "es6/*": "javascripts/es6/*.js",
+    "bootstraps/*": "javascripts/bootstraps/*.js",
```

Comment:
I know I discussed whether it belonged in `package.json` with @guybedford in some PR a while back but I can't remember the outcome, I think it may have been to change it here.